File: framework/ecommerce-framework-api/src/main/java/com/sdl/ecommerce/api/edit/MenuItem.java
package com.sdl.ecommerce.api.edit;

/**
 * In-Context Menu Item
 *
 * @author nic
 */
public class MenuItem {

    private String title;
    private String url;

    public MenuItem(String title, String url) {
        this.title = title;
        this.url = url;
    }

    public String getTitle() {
        return title;
    }

    public String getUrl() {
        return url;
    }
}

File: dxa.net/ecommerce-framework-dotnet/SDL.ECommerce.Rest/Model/MenuItem.cs
using SDL.ECommerce.Api.Model;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SDL.ECommerce.Rest.Model
{
    public class MenuItem : IMenuItem
    {
        public string Title { get; set; }
        public string Url { get; set; }
    }
}
